packages/core/src/lib/utils/index.ts

```diff
@@ -1,7 +1,295 @@
+    totalPages: block.totalPages,
+    blocks: block.blocks.map(
+      (b: ValidatorCompleteBlockType): CompleteBlockType => ({
+        blockHash: b.blockHash,
+        timestamp: b.ts,
+        totalNumberOfTxns: b.totalNumberOfTxns,
+        transactions: b.transactions.map(
+          (t: ValidatorCompleteTxResponse): CompleteTxResponse => ({
+            hash: t.txnHash,
+            fee: t.txnDataAsJson.tx.fee,
+            salt: t.txnDataAsJson.tx.salt,
+            apiToken: t.txnDataAsJson.tx.apitoken,
+            timestamp: +t.ts,
+            category: t.category,
+            from: t.from,
+            recipients: t.recipients.recipients.map((r) => r.address),
+            data: new TextDecoder().decode(
+              new Uint8Array(Buffer.from(t.txnData, 'hex'))
+            ),
+            signature: t.sig,
+          })
+        ),
+      })
+    ),
+  };
+}
+
+export function toSimplifiedBlockResponse(
+  blockResponse: CompleteBlockResponse
+): BlockResponse {
+  return {
+    totalPages: blockResponse.totalPages,
+    lastTimestamp: blockResponse.lastTimestamp,
+    blocks: blockResponse.blocks.map(
+      (b: CompleteBlockType): BlockType => ({
+        blockHash: b.blockHash,
+        timestamp: b.timestamp,
+        totalNumberOfTxns: b.totalNumberOfTxns,
+        transactions: b.transactions.map(
+          (t: CompleteTxResponse): TxResponse => ({
+            hash: t.hash,
+            fee: t.fee,
+            timestamp: t.timestamp,
+            category: t.category,
+            from: t.from,
+            recipients: t.recipients,
+            data: t.data,
+            signature: t.signature,
+          })
+        ),
+      })
+    ),
+  };
+}
+
+export class Utils {
+  static account = {
+    toUniversal(chainAgnosticAddress: string): UniversalAccount {
+      return Utils.toUniversal(chainAgnosticAddress);
+    },
+    toChainAgnostic(universalAccount: UniversalAccount): string {
+      return Utils.toChainAgnostic(universalAccount);
+    },
+  };
+
+  static block = {
+    serialize(block: GeneratedBlock): Uint8Array {
+      return Utils.serializeBlock(block);
+    },
+    deserialize(block: Uint8Array): GeneratedBlock {
+      return Utils.deserializeBlock(block);
+    },
+  };
+
+  static tx = {
+    serialize(tx: Transaction): Uint8Array {
+      return Utils.serialize(tx);
+    },
+    deserialize(tx: Uint8Array): Transaction {
+      return Utils.deserialize(tx);
+    },
+    serializeData(
+      txData: InitDid | InitSessionKey,
+      category: TxCategory
+    ): Uint8Array {
+      return Utils.serializeData(txData, category);
+    },
+    deserializeData(
+      txData: Uint8Array,
+      category: TxCategory
+    ): InitDid | InitSessionKey {
+      return Utils.deserializeData(txData, category);
+    },
+  };
+
+  /**
+   * @param chainAgnosticAddress in CAIP10 - example: eip155:1:0xabc...
+   */
+  private static toUniversal(chainAgnosticAddress: string): UniversalAccount {
```

Comment:
We can have a "CAIP NETWORK" : "CHAIN" mapping in config for this rather than using else if

"eip155"  : "ETHEREUM" 
"solana"  : "SOLANA"

Also can have a 'caip_network`:chainId to string mapping
eg
"eip155" : { "1" : "ETH_MAINNET" }